scripts/static-build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -ue
+
+LC_ALL=C
+
+cd $(dirname "$0")/..
+
+## Run build in container
+
+tar c $(git ls-files) | \
+  docker run --rm -i \
```

Comment:
Did you have a specific reason for not using a standard "bind mount" here? (I mean, a `docker run -v "$PWD:$PWD" -w "$PWD"`)

Anyway, the current version of this `docker-run` command is OK for me, I guess it is fine to provide more isolation and only share the _build/install/default/bin folder we are interested in.

Though, I'd suggest to tweak the first `tar` command to cope with potential files containing whitespace or so (which is not the case today but might occur some day), and maybe add the `set -o pipefail` so that if the `docker run --rm -i …` command fails but the final `tar vx` does not fail, with this option we are sure to "detect" the failure:
```suggestion
set -o pipefail
git ls-files -z | xargs -0 tar c | \
  docker run --rm -i \
```
WDYT?

Reply:
Bind-mounts are terrible as soon as your UIDs and GIDs don't match between the host of the container, which is almost always the case, because there is no automatic mapping taking place. Docker advises to use built-in "named volumes" to work around that, but then the data remains hidden somewhere below `/lib` and you need specific commands to extract it. For a simple case like here, this hack seems much nicer ;)

Your suggestion is indeed better in terms of robustness, thanks !